src/juggleApp/learnToJuggle/LearnToJuggleCard.jsx

```diff
@@ -0,0 +1,137 @@
+      "The highpoint (pinnacle) of your arc should happen near the center of your chest",
+      "Toss it back to your dominant-hand",
+    ],
+  },
+  {
+    numOfBalls: 2,
+    header: "Moving on up",
+    steps: [
+      "Start with 1 ball in each hand",
+      "Toss the ball in your dominant-hand to your off-hand",
+      "Just before the ball reaches the pinnacle of your arc..",
+      "toss the ball in your off-hand to your dominant-hand",
+      "This second ball should follow the same arc as primary ball with just slightly lower so they do not collide",
+      "Be sure to practice starting from your non-dominant hand as well",
+    ],
+  },
+  {
+    numOfBalls: 3,
+    header: "Now with 3 balls",
+    steps: [
+      "Start with 2 balls in your dominant-hand and 1 in your off-hand",
+      "You will begin with the same process as the previous step",
+      "Toss ball in dominant-hand, wait for pinnacle, toss ball in off-hand",
+      "When the second ball is nearing its pinnacle, toss your last ball",
+      "*Note you will catch the first toss before you throw your last throw",
+      "Stop throwing after this third throw- congratulations you have just flashed the cascade!",
+      "Be sure to practice starting from your non-dominant hand as well",
+    ],
+  },
+  {
+    numOfBalls: 3,
+    header: "Three-ball cascade",
+    steps: [
+      "Put it all together, adding 1 additional throw at a time",
+      "You will do the same thing as in the previous step, but instead of stopping you will continue the pattern",
+      "1, 2, 3, 4 stop. 1, 2, 3, 4, 5 stop. Build and build",
+    ],
+  },
+  {
+    numOfBalls: 3,
+    header: "Next Steps",
+    steps: [
+      "Focus on consistent throws, it is a lot easier to catch the ball if it lands in the same spot each time",
+      "Learn some tricks, overthrow and jugglers tennis are great places to begin",
+    ],
+  },
+];
+
+export default function LearnToJuggleCard({
+  step = "0",
+  handleNext,
+  handlePrev,
+}) {
+  return (
+    <Card
+      sx={{
+        margin: { xs: "0", md: "12px auto 0" },
+        height: { xs: "80vh", md: "500px" },
+        width: { xs: "100%", md: "600px" },
+      }}
+    >
+      <CardContent
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          height: "100%",
+        }}
+      >
+        <Typography variant="h6" gutterBottom sx={{ display: { md: "none" } }}>
+          {steps[step].header}
+        </Typography>
+        <Typography variant="h5" color="text.secondary" gutterBottom>
+          Number of Balls: {steps[step].numOfBalls}
+        </Typography>
+        <List
+          sx={{
+            flexGrow: "3",
+            padding: { xs: "12px 0", md: "12px 24px" },
+            width: "100%",
+          }}
+        >
+          {/* some bad naming right here.. steps step steps step */}
+          {steps[step].steps.map((step, i) => (
```

Comment:
I like how you set this up to make it applicable to more learn to juggle stuff in the future, nice